lib/AST/ASTImporter.cpp

```diff
@@ -8075,6 +8092,11 @@ Expected<Decl *> ASTImporter::Import(Decl *FromD) {
     return make_error<ImportError>(*Err);
   }
 
+  // We could import from the current TU without error.  But previously we
+  // already had imported a Decl as `ToD` from another TU and with an error.
```

Comment:
Probably use "ASTImporter" instead of "TU". ("We could import with the current ASTImporter ...")

Reply:
Ok.